### PR TITLE
Fixed issue #232 Added card type for source creation

### PR DESCRIPTION
**Pull request by giacomoParisi**
Fixed issue #232

- Added 'card' source type for stripe source creation

### CODE REVIEW

File: src/utils/types.js [hunk 1]
```diff
@@ -116,4 +116,22 @@ export const createSourceWithParamsPropType = {
   postalCode: PropTypes.string,
   country: PropTypes.string,
   card: PropTypes.string,
+  number: PropTypes.string,
```
Comment (isnifer) [inline src/utils/types.js]: why card params described on the top level?

**Comment by isnifer** [comment]
@giacomoParisi update README, write an example

File: src/utils/types.js (hunk 1 above)
Comment (giacomoParisi) [reply src/utils/types.js]: I noticed that there were already other parameters related to the payment method, such as 'iban' or 'bank'

**Comment by Dallas62** [comment]
Any update on this?
Blocking...

**Comment by isnifer** [comment]
@giacomoParisi @Dallas62 need to update README, need to add test

File: src/utils/types.js [hunk 2]
```diff
@@ -116,4 +116,22 @@ export const createSourceWithParamsPropType = {
...
+  expMonth: PropTypes.number,
+  expYear: PropTypes.number,
+  cvc: PropTypes.string,
+  name: PropTypes.string,
+  addressLine1: PropTypes.string,
```
Comment (isnifer) [inline src/utils/types.js]: there is already `addressLine1` prop-type on line `114`

**Comment by rvignesh89** [comment]
@giacomoParisi @isnifer any chance this can be pushed soon? Willing to help out if needed as this would simplify things for me!

**Comment by iagorm** [comment]
Any update?

**Comment by cybergrind** [comment]
It should have new tests, pass existing tests and be rebased for merge.

**Comment by LRNZ09** [comment]
@isnifer I'm working on this, unfortunately the job `PODSPEC_IOS` failed with a timeout: `The job exceeded the maximum time limit for jobs, and has been terminated. `

**Comment by LRNZ09** [comment]
@isnifer I'm now working on this, really needed on my end

**Comment by cybergrind** [comment]
@LRNZ09 looks good. But still, needs tests for the new functionality

**Comment by LRNZ09** [comment]
@cybergrind what kind of tests do you need? I've seen there's already one which cycles over all types but I think it's not enough

**Comment by cybergrind** [comment]
@LRNZ09 I believe we need to explicitly check that is works with new params and new source type.

**Comment by LRNZ09** [comment]
@cybergrind I've added some tests

**Comment by cybergrind** [comment]
@LRNZ09 looks great. Could you please also add e2e test for it here https://github.com/tipsi/tipsi-stripe/tree/master/example/__tests__

**Comment by cybergrind** [comment]
@LRNZ09 awesome. Thank you for the contribution! Going to release it soon

**Pull request merged**